HW3/app.py

```diff
@@ -0,0 +1,88 @@
+from flask import Flask, render_template_string, request
+from faker import Faker
+import csv
+import requests as rqvsts
+from http import HTTPStatus
+from webargs import fields, validate
+from webargs.flaskparser import use_args
+
+app = Flask(__name__)
+faker = Faker()
+
+# dictionary validate parameters
```

Comment:
for each view should be a separate args list. Better to have a few small interfaces than one big interface.